astroquery/alma/core.py

```diff
@@ -212,6 +212,91 @@ def _gen_sql(payload):
     return sql + where
 
 
+#
+# Authentication session information for connecting to an OIDC instance.  Assumes an OIDC system like Keycloak
+# with a preconfigured client called "oidc".
```

Comment:
In spite of the name and the description, this class only checks that the user/password is valid with any of the Keycloak systems deployed in a list of URLs. Nothing else from the key cloak functionality (such as tokens) are further used in the scheme (but maybe it should?). 

This confused me a great deal as I was trying to figure out what kind of magic Keycloak employs to facilitate authentication on the services so I suggest adding the above paragraph or something to that extend to make it clear and prevent future confusion.